Makefile

```diff
@@ -275,6 +283,19 @@ generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 		output:rbac:dir=$(RBAC_ROOT) \
 		rbac:roleName=manager-role
 
+.PHONY: generate-api-docs
+generate-api-docs: $(GEN_CRD_API_REFERENCE_DOCS) ## Generate api documentation
+	$(GEN_CRD_API_REFERENCE_DOCS) \
+		-api-dir=./api/v1alpha7 \
+		-config=./docs/book/gen-crd-api-reference-docs/config.json \
```

Comment:
This files seems to be missing and I see the following error in the [test results](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-provider-openstack/1702/pull-cluster-api-provider-openstack-test/1709843986627170304):

```
 	-template-dir=./docs/book/gen-crd-api-reference-docs/template \
	-out-file=./docs/book/src/api/v1alpha6/api.md
F1005 08:16:40.577071   16819 main.go:123] failed to open config file: open ./docs/book/gen-crd-api-reference-docs/config.json: no such file or directory
make[1]: *** [Makefile:288: generate-api-docs] Error 255
make[1]: Leaving directory '/home/prow/go/src/sigs.k8s.io/cluster-api-provider-openstack'
make: *** [Makefile:255: generate] Error 2 
```

Do we have to provide it or would the tool fall back to reasonable defaults if we don't?

Reply:
Oh! I understand why CI was failing ... `config.json` is gitignored ! Fixing this !